.github/workflows/windows-tests.yml

```
@@ -32,7 +32,9 @@ jobs:
            python-version: ${{ matrix.PYTHON_VERSION }}
       - name: Install package dependencies
         shell: bash -l {0}
-        run: conda env update --file requirements/environment.yml
+        run: |
+          conda env update --file requirements/environment.yml
+          pip install https://github.com/jupyter/jupyter_client/archive/master.zip --force-reinstall
```

In digging into this a bit, I found that the conda version of `pywin32` was `228` while the version added during the reinstall was `301`.  I found that if I apply the following change, the [Windows tests start passing](https://github.com/kevin-bates/qtconsole/actions/runs/1102574719):
```suggestion
          pip install https://github.com/jupyter/jupyter_client/archive/master.zip --force-reinstall
          pip install --upgrade pywin32==228
```
I don't know if there's some incompatibility with other "pieces" installed via conda then whacked by the reinstall, but it seems like reverting the `pywin32` version to what conda laid down does some good.

My packaging-fu isn't sufficient to troubleshoot this further, but this might allow this PR to move forward.

---

Are there new dependencies in jupyter-client 7? If not, I suggest to install it from master with `--no-deps`. If yes, is it possible to install the new dependencies with conda?

There shouldn't be problems by reinstalling pywin32 with pip, as @kevin-bates suggests, because pywin32 doesn't have dependencies (if I recall correctly). But better to avoid it if possible.

---

7.0 adds a dependency on entrypoints.

---

I'll try adding `conda install entrypoints`

---

Ok, since that's present in Anaconda, please run

    conda install entrypoints

before the `pip install` command that installs jupyter-client from master, then pass `--no-deps` to it. That should solve the problem.

---

Thanks @ccordoba12 - that indeed is a better approach.  I had to also add `nest-asycio` as well: https://github.com/kevin-bates/qtconsole/actions/runs/1103052025

I'll update the within the review.